browser-interface/packages/shared/comms/adapters/voice/liveKitVoiceHandler.ts

```diff
@@ -385,12 +385,14 @@ export const createLiveKitVoiceHandler = async (room: Room): Promise<VoiceHandle
       return validInput
     },
     async destroy() {
-      room.localParticipant.unpublishTracks(
-        Array.from(room.localParticipant.audioTracks.values())
-          .map(($) => $.audioTrack!)
-          .filter(Boolean)
-      )
-      await handleDisconnect()
+      await Promise.all([
```

Comment:
I think we may face some unknown livekit race conditions here. I'd play safe with `allSettled` instead of `all`
```suggestion
      await Promise.allSettled([
```